src/docs/product/sentry-basics/search/searchable-properties/index.mdx

```diff
@@ -0,0 +1,1158 @@
+sidebar_order: 10
+description: "Learn more about searchable issue and event properties."
+---
+
+Sentry's search provides you with reserved keywords, like `is`, `user`, `server`, and `browser`, that you can use to search on the properties of issues and events (as well as the special case of releases). You can also create [custom tags](#custom-tags) on which to search. This page includes [a canonical list of all available issue and event search properties](#list-of-all-available-properties) as well as guidance for how to use these properties.
+
+## Search Properties
+
+Sentry's searchable properties fall into one of four categories: event, issue, session replay, or release.
+
+### Issue Properties
+
+<Include name="searchable-properties/intro-for-issues.mdx" />
+
+- [List of searchable properties for issues](/product/sentry-basics/search/searchable-properties/issues/)
+
+### Event Properties
+
+<Include name="searchable-properties/intro-for-events.mdx" />
+
+- [List of searchable properties for events](/product/sentry-basics/search/searchable-properties/events/)
+
+### Session Replay Properties
+
+<Include name="searchable-properties/intro-for-session-replay.mdx" />
+
+- [List of searchable properties for session replay](/product/sentry-basics/search/searchable-properties/session-replay/)
+
+### Release Properties
+
+<Include name="searchable-properties/intro-for-releases.mdx" />
+
+- [List of searchable properties for releases](/product/sentry-basics/search/searchable-properties/releases/)
+
+## Search Tips
+
+Search terms should auto-complete, and when they don't that means you have an invalid dataset. If you enter an invalid search term and tap Return or Enter, an error message is displayed. For example, in **Discover**, you cannot search using a grouping function (properties with a parentheses in the key, for example `epm()`) if you don't already have a grouping function included in the "Results" table columns.
+
+When you search using a property that's a function, if the key includes a parameter, you still need to add a filter or value to have a complete search token. For example, the function `count_unique(field)` takes whichever field you choose as the parameter and then you add a number as your value to filter on. So a typical search might look like: `count_unique(user):>=20`.
+
+Other things to note:
+
+- Search properties that are duration or number types are typically used with a [comparison operator](/product/sentry-basics/search/#comparison-operators) rather than just a colon (`:`) to find exact matches, as an exact match is unlikely to exist.
+- Properties with the notation "Doesn't take a parameter" still require a filter or value, but don't take a parameter inside the parentheses. For example, `epm():>12`.
+- Default fields/parameters in search functions can be updated. For example, the default parameters `column`, `operator`, and `value` can be updated for `count_if(column,operator,value)`.
+
+## List of All Available Properties
```

Comment:
Is there a reason to list all of these here and in separate pages for events, issues, replays, releases? Would it make sense to just remove this section?

Reply:
Good that you bring it up! I will remove the combined list as I also think this only made documentation easy for us (so far), but for users it does not provide any value.